assets/dev/js/editor/regions/responsive-bar/view.js

```diff
@@ -0,0 +1,100 @@
+export default class View extends Marionette.ItemView {
+	getTemplate() {
+		return '#tmpl-elementor-templates-responsive-top-bar';
+	}
+
+	className() {
+		return 'e-mq-bar';
```

Comment:
`mq`? why not a full name?

Reply:
For the same reason [other abbreviations](https://www.abbreviations.com/acronyms/WEBSITES) exist.

If you think it should be changed to something else, just tell me what name it should be and I will refactor.

Reply:
The feature is called Responsive Bar. so `rb` will be more understood.

Reply:
The feature name will be `responsive-bar`